build.gradle

```diff
@@ -191,6 +191,12 @@ subprojects {
     //Required for building on Travis' container-based infrastructure to not be killed with
     //'exit code 137' - https://github.com/travis-ci/travis-ci/issues/5582
     jvmArgs '-Xmx512m'
+    if (rootProject.ext.javaVersion >= 17) {
+      jvmArgs(
+        //Required since jacoco does not do that anymore: Agent should not open java.lang package to unnamed module of the application class loader (https://github.com/jacoco/jacoco/pull/1334).
```

Comment:
The comment explains this PR but not really why we need the `add-opens` for our projects. And it is not required for every [single one](https://ge.spockframework.org/s/hle6g33s6a2la/tests/overview?collapse-all), I'd prefer to only add it where we need it for now with a reference as to why.

Reply:
I have moved to jvmArgs into the required projects: Spring, Guice and cglib test.
The root cause is, that all of them are using cglib under the hood.

Reply:
Much better thanks